resqpy/grid/grid_functions.py

```diff
@@ -0,0 +1,187 @@
+"""Submodule containing some general grid functions."""
+
```

Comment:
Let's name this module something else to avoid confusion with olio/grid_functions.py